cmd/api/README.md

```diff
@@ -11,6 +11,8 @@
 | DB_SSLMODE               | Database SSL mode                                                                                                                 | verify-full                                  |
 | DB_SSLROOTCERT           | Path to CA cert used to validate Database cert                                                                                    | /etc/tls/db/ca.crt                           |
 | DB_ENABLE_AUTO_MIGRATION | Auto-migrate the database on startup (create/update schemas). For further details, refer to <https://gorm.io/docs/migration.html> | true (default)                               |
+| DB_MAX_IDLE_CONNECTIONS  | The number of idle database connections to keep open                                                                              | 10                                           |
```

Comment:
Can you also add this to https://github.com/tektoncd/results/blob/main/config/base/env/config ?

Reply:
in case you are now sidetracked / distracted by your day job @stuartwdouglas I will cherrypick this commit here into my upcoming PR for exposing the k8s client tuning and grpc worker thread tuning , add this ask from @khrm,  and we can go from there

thanks again for the contribution

Reply:
I have https://github.com/tektoncd/results/pull/744 now

Reply:
I guess this can be closed now then?